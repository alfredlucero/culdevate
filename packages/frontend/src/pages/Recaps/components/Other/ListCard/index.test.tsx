import React from "react";
import { render } from "@testing-library/react";
import OtherListCard from "./index";

describe("<OtherListCard />", () => {
  test("should render without error", () => {
    const { container } = render(
      <OtherListCard
        onClickAdd={() => {}}
        onClickView={() => {}}
        count={10}
        className="extra-list-class"
        testId="testId"
      />,
    );

    expect(container).toMatchSnapshot();
  });
});
