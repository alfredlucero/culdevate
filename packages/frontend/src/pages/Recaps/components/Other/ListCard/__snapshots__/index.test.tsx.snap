// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OtherListCard /> should render without error 1`] = `
<div>
  <div
    class="bg-white shadow-md rounded px-8 py-6 w-64 extra-list-class"
    data-testid="testId"
  >
    <div
      class="mb-6 flex justify-center"
    >
      <div
        class="flex justify-center items-center p-3 rounded-lg bg-gray-200 recap-icon"
      >
        <div
          class="font-sans text-3xl text-teal-400"
          id=""
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-highlighter fa-w-17 "
            data-icon="highlighter"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 544 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 479.98L99.92 512l35.45-35.45-67.04-67.04L0 479.98zm124.61-240.01a36.592 36.592 0 0 0-10.79 38.1l13.05 42.83-50.93 50.94 96.23 96.23 50.86-50.86 42.74 13.08c13.73 4.2 28.65-.01 38.15-10.78l35.55-41.64-173.34-173.34-41.52 35.44zm403.31-160.7l-63.2-63.2c-20.49-20.49-53.38-21.52-75.12-2.35L190.55 183.68l169.77 169.78L530.27 154.4c19.18-21.74 18.15-54.63-2.35-75.13z"
              fill="currentColor"
            />
          </svg>
        </div>
      </div>
    </div>
    <h4
      class="font-mono font-bold text-gray-800 text-xl mb-4 text-center"
    >
      Other
    </h4>
    <div
      class="flex justify-center mb-4"
    >
      <button
        class="font-bold py-2 px-4 rounded bg-transparent text-teal-700 border border-teal-500 hover:bg-teal-500 hover:text-white hover:border-transparent mr-2"
        type="button"
      >
        <div
          class="flex items-center"
        >
          View
        </div>
      </button>
      <button
        class="font-bold py-2 px-4 rounded font-bold py-2 px-4 rounded bg-teal-500 text-white hover:bg-teal-700 "
        type="button"
      >
        <div
          class="flex items-center"
        >
          Add
        </div>
      </button>
    </div>
    <div
      class="flex justify-end"
    >
      <div
        class="rounded bg-gray-200 text-gray-700 text-sm p-2 w-8 flex justify-center items-center"
      >
        10
      </div>
    </div>
  </div>
</div>
`;
